### ISSUE TITLE
HTLB for Deck Not Working with Decky Loader 2.10.15-pre2

### ISSUE THREAD

**Issue by matthewclaussen** [hulkrelax/hltb-for-deck#28]
Steam OS 3.5.15 preview update broke Decky Loader 2.10.14. Decky Loader was updated to 2.10.15-pre2 and sadly the HLTB plugin no longer works.

**Comment by shigutso** [comment]
Same issue here. All my other plugins are working

**Comment by Mobeeuz** [comment]
No, Wine Cellar and Storage Cleaner are also broken with the exact same issue...

**Comment by SimPilotAdamT** [comment]
Same issue here but it's slightly different. It's actually caused by the Steam Client beta.

In developer options, make sure advanced update channels is enabled, so that your cab still use OS beta versions. Then in system settings, make sure you're using the steam client stable version, not the beta one.

If you're affected by [this bug](https://github.com/ValveSoftware/steam-for-linux/issues/10494), do whatever you will, it depends on how you want to play. Idm those plugins being unavailable temporarily, so long as we can get a fix rolled out soon. Could it be a decky bug?

I use Decky v2.10.16-pre1

**Comment by SimPilotAdamT** [comment]
> No, Wine Cellar and Storage Cleaner are also broken with the exact same issue...

Storage cleaner seems to be fixed now, at least in the pre release decky store repo. Not sure about wine cellar, I don't use that one.

**Comment by dtantono** [comment]
now with SteamOS get new version. this error comes up on latest decky loader

**Comment by dnet890** [comment]
https://store.steampowered.com/news/app/1675200/view/4124805233776242150 the new update comes and it still have same issues
